contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics/process.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import psutil
+
+from opencensus.metrics.export.gauge import DerivedLongGauge
+from opencensus.metrics.export.gauge import DerivedDoubleGauge
+
+logger = logging.getLogger(__name__)
+PROCESS = psutil.Process()
+
+
+class ProcessMemoryMetric(object):
+    NAME = "\\Process(??APP_WIN32_PROC??)\\Private Bytes"
+    @staticmethod
+    def get_value():
+        try:
+            return PROCESS.memory_info().rss
+        except Exception:
+            logger.exception('Error handling get process private bytes.')
+
+    def __call__(self):
+        """ Returns a derived gauge for private bytes for the current process
+
+        Private bytes for the current process is measured by the Resident Set
+        Size, which is the non-swapped physical memory a process has used.
+
+        :rtype: :class:`opencensus.metrics.export.gauge.DerivedLongGauge`
+        :return: The gauge representing the private bytes metric
+        """
+        gauge = DerivedLongGauge(
+            ProcessMemoryMetric.NAME,
+            'Amount of memory process has used in bytes',
+            'byte',
+            [])
+        gauge.create_default_time_series(ProcessMemoryMetric.get_value)
+        return gauge
+
+
+class ProcessCPUMetric(object):
+    NAME = "\\Process(??APP_WIN32_PROC??)\\% Processor Time"
+    CPU_COUNT = psutil.cpu_count(logical=True)
+    @staticmethod
```

Comment:
Would it be better to make this a `@classmethod` then use `cls.CPU_COUNT` instead of hardcoding the class name again?

Reply:
Now looking more into when to use `@classmethod` I think it will be a good idea to use that instead of `@staticmethod` for all of the metrics, since `get_value` will be working with class specific data (and not isolated functions like `@staticmethod` is normally used for).